CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+      "2": [
+        26
+      ],
+      "3": [
+        "Ed25519"
+      ],
+      "4": 12345,
+      "5": "https://oracle.tokenproject.io/",
+      "6": {
+        "26": {
+          "1": [
+            {
+              "0": [
+                "d894897411707efa755a76deb66d26dfd50593f2e70863e1661e98a0",
+                "7370616365636f696e73"
+              ],
+              "1": "spacecoins",
+              "2": "SPACE",
+              "3": [
+                "the OG Cardano community token",
+                "-",
+                "whatever you do, your did it!",
+                "",
+                "Learn more at https://spacecoins.io!"
+              ],
+              "4": 0,
+              "5": [
+                "https://",
+                "spacecoins.io"
+              ],
+              "6": [
+                "ipfs://",
+                "bafkreib3e5u4am2btduu5s76rdznmqgmmrd4l6xf2vpi4vzldxe25fqapy"
+              ],
+              "7": [
+                [
+                  "ipfs://",
+                  "bafkreibva6x6dwxqksmnozg44vpixja6jlhm2w7ueydkyk4lpxbowdbqly"
+                ],
+                "3507afe1daf05498d764dce55e8ba41e4acecd5bf42606ac2b8b7dc2eb0c305e"
+              ],
+              "8": [
+                "37da68d092f4e61feb237de5e86b404171b59d3880d340023a16a6983491736d",
+                "0"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "2": [
+      [
+        "e97316c52c85eab276fd40feacf78bc5eff74e225e744567140070c3j",
+        [
+          "witness may be broken up into a sub-array of 64-character strings",
+          "assuming that the overall length of the witness is longer than the",
+          "64-character on-chain metadata limit for strings."
+        ]
+      ]
+    ]
+  }
+}
+```
+
+## Rationale
+
+For this specification, I have drawn inspiration from 
+[CIP-36: Catalyst/Voltaire Registration Metadata Format](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0036)
+which succinctly and canonically publishes data to the main chain (L1) via a metadata transaction and without any
+required modification or customization to the underlying ledger with regard to Native Assets.
+
+By leveraging the existing signing keys present in native asset scripts from the beginning of the Mary Era on Cardano we
+can enable all projects to update and provide additional, verified information about their project in a canonical, 
+verifiable, and on-chain way while also providing for additional off-chain information.
+
+This makes this CIP backwards compatible with all existing standards (CIP-25, 26, 27, 68, etc) while also providing the
+flexibility for future-proofing and adding additional context and information in the future as additional use cases,
+utility, and standards evolve.
+
+## Path to Active
+
+### Acceptance Criteria
+
+Ideally this CIP would receive feedback, criticism, and refinement from the community of people involved in and with
```

Comment:
Let's remove "Ideally" and make it an actual requirement for activation.

The proposal shall also be adopted in major explorers and/or wallets; as well as adopted by various tokens projects.